### PR TITLE
Usagov 430 spanish directories

### PR DESCRIPTION
**Pull request by IsabelLaurenceau**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-403

## Description

* Changes
To implement Spanish Directories the same view **Federal Agencies** is used but a secondary block **Indice Agencias A-Z** with *Content: Translation language set to Español* and a secondary **Alpha List ES** also with translation set to Español. Both the  **Federal Agencies A-Z** and **Indice Agencias A-Z** blocks have the link to standalone page text changed from `<a href="{{ view_node }}">More information about {{ title }}&nbsp;></a>` to only `{{ view_node }}`. This allows the rest of the link text to be set in **views-view-fields.html.twig** with the correct language following the pattern for the rest of the fields.

* The first implementation of Spanish Directories created a separate view. This methodology worked however it required  three additional Twig files (*views-view-summary-Spanish.html.twig, views-view-list-Spanish.html.twig, views-view-fields-Spanish.html.twig*) and two additional configuration files. To reduce repeat code the decision was made to utilize the same View with duplicate block. This allowed the original (*views-view-summary.html.twig, views-view-list.html.twig, views-view-fields.html.twig*) with a language check.

* Using only one block was considered for both languages however to implement this would then require a supplemental pre-process hook to only retrieve the agencies and Alpha List of the selected language. This would actually introduce more code into the repository to be maintained than the current implementation with two blocks. Though the block and alpha list or essentially duplicates of the English ones this allows them to share the same three Twig files for better maintainability.

* To Test 
    * Import config changes

    * Import federal directories following https://github.com/usagov/usagov-2021/blob/dev/web/modules/custom/usagov_directories/docs/Importing_Federal_Agency_Records.md 

    * Create a new basic page 
        * Title- Agencias federales
        * "Español" selected for language
        * URL alias- /indice-agencias
        
Spanish agencies should be displayed on the page. 
Headings in accordion should be in Spanish
Letter Indexes should take you to correct Spanish query url 
Previous and Back buttons should also  take you to correct Spanish query url 

## Checklist for the Developer

- [x] A link to the JIRA ticket has been added above.
- [x] The JIRA ticket identifies the desired result of this change.
- [x] The JIRA ticket contains clear acceptance criteria.
- [x] The JIRA ticket contains clear testing steps.
- [x] The JIRA ticket contains a description of "Done".
- [x] Any preparation/installation/update steps required for this code to work properly (drush commands, scripts, configuration updates) are documented in the ticket.

## Checklist for the Peer Reviewers
- [x] The branch name of this PR matches the project standards.
- [ ] QA steps are followed and any changes to process are updated in the ticket.
- [ ] Code Standards are followed, there are no bad practices in use.
- [ ] Config changes (if any) include only necessary changes.
- [ ] No errors in Drupal or Client Browser.
- [ ] Code has been tested locally (if possible).
- [ ] Following AC and Testing Steps verify changes work as expected.
- [ ] There are no known side-effects outside of expected behavior.

### CODE REVIEW

File: web/themes/custom/usagov/templates/views-view-fields--federal-agencies.html.twig [hunk 1]
```diff
@@ -55,36 +55,39 @@
...
+      <p>{{ lang == 'en' ? 'Find an office near you' : 'Encuentre una oficina cercana' }}:&nbsp;</p>
       <p>{{- fields.field_offices_near_you.content -}}</p>
     </div>
     {% endif %}
 
     {% if fields.nothing.content|striptags|trim %}
-    <div>
-      <p>{{- fields.nothing.content -}}</p>
-    </div>
+      <div>
```
Comment (akf) [inline web/themes/custom/usagov/templates/views-view-fields--federal-agencies.html.twig:85]: Nice! Glad to have this text out of the view and into a twig file.

**Review by akf — CHANGES_REQUESTED** [review]
I have a change request to your setup, which will require adjustment (probably to the block layout): 
The basic page "Agencias federales" should have: 
*  "Español" selected for language
* URL alias entered as simply "/indice-agencias" 
Our Drupal setup automatically prepends "/es" to the paths of Spanish-language pages, so if I follow your directions and choose Español, I get a page at /es/es/indice-agencias. If I don't choose Español, other things are wrong. If I choose Español and enter /indice-agencias, the page at /es/indice-agencias doesn't have the index on it.

**Comment by IsabelLaurenceau** [comment]
> I have a change request to your setup, which will require adjustment (probably to the block layout): The basic page "Agencias federales" should have:
> 
> * "Español" selected for language
> * URL alias entered as simply "/indice-agencias"
>   Our Drupal setup automatically prepends "/es" to the paths of Spanish-language pages, so if I follow your directions and choose Español, I get a page at /es/es/indice-agencias. If I don't choose Español, other things are wrong. If I choose Español and enter /indice-agencias, the page at /es/indice-agencias doesn't have the index on it.

Ah okay, I did not know that Drupal did that for us! And you were right the only change needed was in the block layout.

**Review by akf — APPROVED** [review]

**Comment by akf** [comment]
Looks good! I realized we will also want to add the menu link, so I added that to the testing instructions in the ticket.

**Pull request merged**